controls/V-73229.rb

```diff
@@ -1,9 +1,9 @@
 control 'V-73229' do
-  title "Manually managed application account passwords must be at least 15
+  title "Manually managed application account passwords must be at least 14
```

Comment:
I would prefer we move any data elements like this to `inputs` so that we can adjust them easily in overlays and dependant profiles.